src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -12,105 +12,325 @@ Sentry's Performance features are currently in beta. For more details about acce
+
+**Transaction Duration Metrics**
+
+The following functions aggregate transaction durations:
+
+- average
+- various percentiles (by default, the pre-built Transactions query shows the 75th and 95th percentiles, but there are many other options, including a custom percentile)
+
+One use case for tracking these statistics is to help you identify transactions that are slower than your organization's target SLAs.
+
+A word of caution when looking at averages and percentiles: In most cases, you'll want to set up tracing so that only [a fraction](#data-sampling) of possible traces are actually sent to Sentry, to avoid overwhelming your system. Further, you may want to filter your transaction data by date or other factors, or you may be tracing a relatively uncommon operation. For all of these reasons, you may end up with average and percentile data that is directionally correct, but not accurate. (To use the most extreme case as an example, if only a single transaction matches your filters, you can still compute an "average" duration, even though that's clearly not what is usually meant by "average.")
+
+The problem of small sample size (and the resulting inability to be usefully accurate) will happen more often for some metrics than others, and sample size will also vary by row. For example, it takes less data to calculate a meaningful average than it does to calculate an equally meaningful 95th percentile. Further, a row representing requests to `/settings/my-awesome-org/` will likely contain many times as many transactions as one representing requests to `/settings/my-awesome-org/projects/best-project-ever/`.
+
+**Transaction Frequency Metrics**
+
+The following functions aggregate transaction counts and the rate at which transactions are recorded:
+
+- count
+- count unique values (for a given field)
+- average requests (transactions) per second
+- average requests (transactions) per minute
+
+Each of these functions is calculated with respect to the collection of transactions within the given row, which means the numbers will change as you filter your data or change the time window. Also, if you have set up your SDK to [sample your data](#data-sampling), remember that only the transactions that are sent to Sentry are counted. So if a row containing transactions representing requests to a given endpoint is calculated to be receiving 5 requests per second, and you've got a 25% sampling rate enabled, in reality you're getting approximately 20 (5 * 4) requests to that endpoint each second.
+
+### Transaction Detail View
+
+When you open a transaction event in Discover (by clicking on the icon at the left side of the row), you'll see the **span view** at the top of the page. Other information the SDK captured as part of the transaction event (such as the transaction's tags and automatically collected breadcrumbs) is displayed underneath and to the right of the span view.
```

Comment:
Yeah, that conversation was me, LOL - I wanted to know if the transaction name was going to be clickable, because I didn't find clicking the icon particularly intuitive, and it took me a sec to figure out how to open up the transaction. The answer I got was that they'd thought about it and decided no, since you can change the order of the columns, and so the icon was going to remain the way in (along with, perhaps, something in the row's context menu).

So yeah, as a matter of principle I totally agree with you - this is also why I'm not super hot on screenshots, because they're nearly impossible to keep up to date. In this case, though, I felt like mentioning the icon was warranted just because it felt so non-obvious to me. What do you think?

Also @dashed @markstory - do you have any thoughts here, since this is your area of expertise?

Reply:
I think this is fine for now. 

Today, the span view can only be viewed in one singular page in the Sentry product; and that is within the Discover details page.

The objective of this paragraph was to briefly describe where developers can find the span view.

Reply:
👍🏻